src/pages/account/logins/index.tsx

```diff
@@ -232,3 +249,24 @@ const Button = styled.button<ButtonProps>`
     color: ${({ theme }) => theme.colors.contrastCopy};
   }
 `;
+
+export const ShowMoreButton = styled.button<ButtonProps>`
+  padding: 0.5rem;
+  margin-right: 1rem;
+  border-radius: ${({ theme }) => theme.radii[0]};
+  background-color: transparent;
+  ${(props) => `color: ${props.color}`};
```

Comment:
This is taken from my previous button. You don't need to pass the color prop any more, just use the theme right away :).

Let me know if this is not clear.

Reply:
OK fixed 0fe5a7d